text/87_interacting_with_stakeholders.md

```diff
@@ -31,7 +31,7 @@ This is only a rough guide to some of the kinds of stakeholders you will interac
 
-One of the key chasms in communication is between technical and non-technical personnel.  If you're reading this book, and I tell you that since you've got a O(n^2) sorting algorithm, you're going to see an exponential increase in RAM usage with larger data sets, you're going to have a pretty good idea of what I mean.  You may even know how to fix it (by quickly searching for "efficient sorting algorithms", of course).  However, this sentence is basically Greek to non-technical personnel (except for fluent speakers Greek, for whom it is some other language that they don't know).  However, this is the way developers, testers, and other technical personnel talk, because it's how you end up thinking when you deal with it all day, every day.  Remember, though, that other people may spend the majority of their day dealing with accounting, or sales, or marketing, or a million other things, and simply do not have the same frame of reference as you do when it comes to the system you are working on.
+One of the key chasms in communication is between technical and non-technical personnel.  If you're reading this book, and I tell you that since you've got a O(n^2) sorting algorithm, you're going to see an exponential increase in RAM usage with larger data sets, you're going to have a pretty good idea of what I mean.  You may even know how to fix it (by quickly searching for "efficient sorting algorithms", of course).  However, this sentence is basically Greek to non-technical personnel (except for fluent speakers of Greek, for whom it is some other language that they don't know).  However, this is the way developers, testers, and other technical personnel talk, because it's how you end up thinking when you deal with it all day, every day.  Remember, though, that other people may spend the majority of their day dealing with accounting, or sales, or marketing, or a million other things, and simply do not have the same frame of reference as you do when it comes to the system you are working on.
```

Comment:
Apparently, the relevant Greek phrase here is "Αυτά μου φαίνονται κινέζικα." ("This strikes me as Chinese.")
https://en.wikipedia.org/wiki/Greek_to_me#In_other_languages